src/autoscaler/eventgenerator/exampleconfig/example.yaml

```diff
@@ -1,9 +1,7 @@
 logging:
   level: "debug"
 server:
```

Comment:
shouldn't be node_count under pool object?

Reply:
fixure is not used, removed in f238e1b54